src/app/lib/ENSUtils.ts

```diff
@@ -1,4 +1,7 @@
 import { ethProvider } from "@/app/lib/provider";
+import { truncateAddress } from "@/app/lib/utils/text";
+
+export const isENSName = (name: string) => name.endsWith(".eth");
```

Comment:
ens name technically do not have to end with .eth
I'd use `isAddress` check from `viem` instead and assume it's ens otherwise
@andreitr